.github/workflows/toolchain-build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       id: get-date
       run: |
         ln -sf /usr/share/zoneinfo/Asia/Kolkata /etc/localtime
```

Comment:
Usage of set-output were in lines 48 and 90, please change those lines as well to reflect the changes of env usage.